cms/toolbar/toolbar.py

```diff
@@ -329,6 +333,22 @@ def request_hook(self):
         else:
             return self._request_hook_post()
 
+    def get_cached_template(self, template):
+        if isinstance(template, Template):
+            return template
+
+        if not template in self._cached_templates:
+            self._cached_templates[template] = get_template(template)
+        return self._cached_templates[template]
+
+    @cached_property
+    def drag_item_template(self):
+        return get_template('cms/toolbar/dragitem.html')
+
+    @cached_property
+    def dragbar_template(self):
+        return get_template('cms/toolbar/dragbar.html')
+
```

Comment:
I thought about it but decided against it because unlike other templates, we already know these before hand and can avoid the extra call to the get_cached_template method every time. I think better would be to use self.get_cached_template() combined with the decorator so we reuse our cached template logic but only call the method once. Ok for you?